examples/golang/components/http-client-tinygo/hello.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"log"
+	"math"
+
+	http "github.com/wasmcloud/wasmcloud/examples/golang/components/http-client-tinygo/gen"
+)
+
+// Helper type aliases to make code more readable
+type HttpRequest = http.ExportsWasiHttp0_2_0_IncomingHandlerIncomingRequest
+type HttpResponseWriter = http.ExportsWasiHttp0_2_0_IncomingHandlerResponseOutparam
+type HttpOutgoingResponse = http.WasiHttp0_2_0_TypesOutgoingResponse
+type HttpError = http.WasiHttp0_2_0_TypesErrorCode
+
+type HttpServer struct{}
+
+func init() {
+	httpserver := HttpServer{}
+	// Set the incoming handler struct to HttpServer
+	http.SetExportsWasiHttp0_2_0_IncomingHandler(httpserver)
+}
+
+func (h HttpServer) Handle(request HttpRequest, responseWriter HttpResponseWriter) {
+	req := http.NewOutgoingRequest(http.NewFields())
+	req.SetScheme(http.Some[http.WasiHttp0_2_0_TypesScheme](http.WasiHttp0_2_0_TypesSchemeHttps()))
+	req.SetAuthority(http.Some("postman-echo.com"))
+	req.SetPathWithQuery(http.Some("/bytes/100/mb"))
```

Comment:
What do you think about cutting this down to a lower amount, like 10kb just so the terminal output is a little more manageable?

Our other example is the dog-fetcher, which could be a good way to give back an output to the user that they can see the result right away

Reply:
sure . i ll change the endpoint.